### DISCUSSION TITLE
Path features not available in v1.2.2

### DISCUSSION THREAD

**Discussion by Gregg8** [Day8/re-frame-10x#351] [General]
@MawiraIke, we have been using `1.2.1` and noticed a newer version, so updated to `1.2.2` overnight. 

Having done that, all the inspector path features are missing in this version. Was this intentional or might it be something to do with my environment?

So we've rolled back to `1.2.1` for the time being.

Looking forward to the next version 👍

**Comment by MawiraIke** [comment]
@Gregg8 path features are still available in `1.2.2`. 

However due to a delay in rendering paths for large db's, we made it optional to render path annotations (default might be false) in version `1.2.2` and the option can be toggled by using this checkbox above the path inspector:

![paths](https://user-images.githubusercontent.com/38689046/156301904-38a35489-7f41-43d8-8122-bbedb0f518d4.jpg)

The CHANGELOG has been updated with the changes but the [PR](https://github.com/day8/re-frame-10x/pull/350/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR20) is still open.

I think this should be the issue.

**Comment by Gregg8** [comment]
Haha, thanks. It was "hiding in plain sight".

Yeah, the default should be on, or is it intended for this checkbox to be temporary, until the performance is improved?

**Comment by MawiraIke** [comment]
I think that is correct. If the speed is improved enough to almost match when rendering paths is false, then the option will be made to default to true or the checkbox removed altogether.
